### DISCUSSION TITLE
Welcome to durexforth Discussions!

### DISCUSSION THREAD

**Discussion by jkotlinski** [jkotlinski/durexforth#315] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Whammo** [comment]
Hello out there!  I've been hacking away at durexForth for many years, it's great fun to use. I contributed LS, and SYS.
The new file i/o words have sparked my interest lately, what was once clumsy for me is now easy to visualize, code and follow.
 I've got lots of pieces of code and plans to integrate them, but it's the contributions of others that keeps me interested.
  Thanks!

**Reply by JoannaK** [reply]
HI, new to durexforth and github. Old to C64, 6502 and Forth, though it's been decades, so don't remember much. I made https://github.com/JoannaK/durexforth-extras repo for storing my code as it is at the moment.
